<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manageOrder</class>
 <widget class="QWidget" name="manageOrder">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Order</string>
  </property>
  <widget class="QPushButton" name="createOrderButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Order</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="menuOrdetTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>731</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="viewOrderButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>View Order</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reserveTableButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Reserve Table</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tableId">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
